lib/engines/hcktest/platforms/schema.json

```diff
@@ -0,0 +1,130 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "type": "object",
+  "properties": {
+    "name": {
+      "type": "string"
+    },
+    "kit": {
+      "type": "string"
+    },
+    "type": {
+      "type": "string",
+      "enum": [
+        "virtual",
+        "physical"
```

Comment:
Let's name them `qemuhck` and `physhck` for consistency too. Especially It's better to avoid the name `virtual` as the specified properties are mostly specific to QEMU and we will want to have another type when we introduce support for e.g., Hyper-V.